test/allowlist/gcc/common.log

```diff
@@ -39,4 +42,6 @@ FAIL: gcc.dg/tree-ssa/ssa-sink-18.c
 FAIL: gcc.dg/debug/btf/btf-datasec-1.c
 # By Jiawei:
 #   https://gcc.gnu.org/git/?p=gcc.git;a=commit;h=dc32901a0221a43e121591b9819b4e33bcc2fd0a
+#   https://gcc.gnu.org/git/?p=gcc.git;a=commit;h=b18e5d7e5f9df69759f0fbc2bed91d5e51313e79
```

Comment:
I think this line should be one line more down (after the const7.C FAIL).

Reply:
Changed, thanks for your suggestion!